python/ray/services.py

```diff
@@ -1341,11 +1369,12 @@ def start_ray_processes(address_info=None,
     raylet_socket_names = address_info["raylet_socket_names"]
 
     # Get the ports to use for the object managers if any are provided.
-    object_manager_ports = (address_info["object_manager_ports"] if
-                            "object_manager_ports" in address_info else None)
     if not isinstance(object_manager_ports, list):
         object_manager_ports = num_local_schedulers * [object_manager_ports]
     assert len(object_manager_ports) == num_local_schedulers
+    if not isinstance(node_manager_ports, list):
+        node_manager_ports = num_local_schedulers * [node_manager_ports]
```

Comment:
Good point, I'll add a check for this. I actually want to get rid of this code path soon so that it isn't possible to simulate multiple nodes this way and that we instead use the cluster_utils that @richardliaw added recently.

Reply:
So, I added a check that when a single value is passed in for the object manager or node manager port, either there is only one object/node manager being started or the value is `None`.